packages/webdoc-template-library/src/template-plugins/LinkerPlugin.js

```diff
@@ -374,15 +374,8 @@ function LinkerPluginShell() {
      * @return {string} uri with siteRoot
      */
     processInternalURI(uri: string, options: { outputRelative?: boolean } = {}): string {
-      if (!options.outputRelative) {
-        uri = uri
-          .replace("%3CsiteRoot%3E", this.siteRoot)
-          .replace("//", "/");// needed if siteRoot = "";
-      } else {
-        uri = uri.replace("/%3CsiteRoot%3E/", "");
-      }
-
-      return uri;
+      return uri.replaceAll("<siteRoot>", options.outputRelative ? (this.siteRoot || "") : "")
```

Comment:
"&lt;siteRoot&gt;" should be in the URI only once so we don't need to use `replaceAll`, which isn't available on Node 12 I think (if you really want global replace, use regex instead).

This is failing the CI.

Reply:
fixed in cb1138e